.github/workflows/ci.yml

```diff
@@ -196,6 +196,78 @@ jobs:
     - name: make test
       run: make test HARNESS_JOBS=${HARNESS_JOBS:-4}
```

Comment:
Should we run some tests with this code and asan?

Reply:
I'll look into that.

Reply:
Done.